source/jsonnet/england-wales/ccs/blocks/who-lives-here/primary_person_list_collector.jsonnet

```diff
@@ -56,13 +56,13 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
             title: '“Usually living at” means:',
           },
           {
-            description: 'The address at which you generally spend most time, for most people this will be their permanent or family home.',
+            description: 'The address at which you generally spend most time. For most people this will be their permanent or family home.',
           },
           {
             list: [
-              '<strong>full-time students</strong> should include themselves here',
-              '<strong>armed forces members</strong> should include themselves at their home address if they have one',
-              '<strong>include anyone temporarily outside the UK, if this was for less than 12 months</strong>',
+              '<strong>Full-time students</strong> should include themselves here.',
```

Comment:
Although this is correct according to the spreadsheet, I am under the impression that list items start with small letters

Reply:
I will double check with @donnap573 on this